vhdbuilder/packer/install-dependencies.sh

```diff
@@ -121,7 +122,7 @@ fi
 
 # Since we do not build Ubuntu 16.04 images anymore, always override network config and disable NTP + Timesyncd and install Chrony
 # Mariner does this differently, so only do it for Ubuntu
-if [[ $OS != $MARINER_OS_NAME ]]; then
+if [[ $OS != $MARINER_OS_NAME ]] && [[ $OS != $AZURELINUX_OS_NAME ]]; then
```

Comment:
same thing here, we'd no longer be doing this on mariner V2-based AzureLinux?

Reply:
`$OS` for the mariner V2-based AzureLinux is `MARINER` from my understanding. The `OS` variable is being [initialized ](https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/install-dependencies.sh#L11) at the top of this file. For the mariner V2-based AzureLinux, I believe `$OS` is MARINER but the [OS_SKU](https://github.com/Azure/AgentBaker/blob/master/.pipelines/.vsts-vhd-builder-release.yaml#L367) is AzureLinux

For example, in the `buildAzureLinuxV2Gen2` [job](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=100391581&view=logs&j=d8a397d3-f8e5-579e-295f-8bdfee45fee5&t=80fc7348-a696-5095-f47e-0c50c06931d7&l=3896), I see `MARINER != MARINER`.

I believe we were not going into this condition before at all for Mariner.

Reply:
gotcha, that's good to know moving forward

Reply:
im assuming this means V3 is the first version where the actual OS rebrand to Azure Linux is reflected?

Reply:
Yeah, that's correct